src/Libraries/RevitNodes/Elements/Views/StructuralPlanView.cs

```diff
@@ -0,0 +1,107 @@
+using System;
+
+using Autodesk.Revit.DB;
+using RevitServices.Persistence;
+using RevitServices.Transactions;
+
+namespace Revit.Elements.Views
+{
+    /// <summary>
+    /// A Revit ViewPlan
+    /// </summary>
+    [DynamoServices.RegisterForTrace]
+    public class StructuralPlanView : PlanView
+    {
+
+        #region Private constructors
+
+        /// <summary>
+        /// Private constructor
+        /// </summary>
+        private StructuralPlanView(Autodesk.Revit.DB.ViewPlan view)
+        {
+            SafeInit(() => InitStructuralPlanView(view));
+        }
+
+        /// <summary>
+        /// Private constructor
+        /// </summary>
+        private StructuralPlanView(Autodesk.Revit.DB.Level level)
+        {
+            SafeInit(() => InitStructuralPlanView(level));
+        }
+
+        #endregion
+
+        #region Helpers for private constructors
+
+        /// <summary>
+        /// Initialize a StructuralPlanView element
+        /// </summary>
+        private void InitStructuralPlanView(Autodesk.Revit.DB.ViewPlan view)
+        {
+            InternalSetPlanView(view);
+        }
+
+        /// <summary>
+        /// Initialize a StructuralPlanView element
+        /// </summary>
+        private void InitStructuralPlanView(Autodesk.Revit.DB.Level level)
+        {
+            TransactionManager.Instance.EnsureInTransaction(Document);
+
+            var vd = CreatePlanView(level, ViewFamily.StructuralPlan);
+
+            InternalSetPlanView(vd);
+
+            TransactionManager.Instance.TransactionTaskDone();
+
+            ElementBinder.CleanupAndSetElementForTrace(Document, this.InternalElement);
+        }
+
+        #endregion
+
+        #region Public static constructors
+
+        /// <summary>
```

Comment:
As a general rule, all publicly accessible constructors, properties, and methods, should have their documentation tags filled out. This will become the documentation that the user sees in the UI. If they are not filled out, the user won't see anything.